src/pages/Page.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from 'react';
+
+import jsonData from '../data.json';
+import Question from '../components/Question';
+import Button from '../components/Button';
+import Loader from '../components/Loader';
+
+const Page = () => {
+  const [question, setQuestion] = useState([]);
+  const [step, setStep] = useState(0);
+
+  useEffect(() => {
+    setQuestion(jsonData);
+  }, []);
+
+  const handleNext = () => {
```

Comment:
use () 
{} is there is more than one action

Reply:
fixed